app/Http/Controllers/DemandController.php

```diff
@@ -196,34 +196,8 @@ public function click(Request $request, $id)
         //     ]);
         // }
 
-        $response = new \Symfony\Component\HttpFoundation\Response($url);
-
-        // last click id will be used to track conversions
-        $response->headers->setCookie(
-            new \Symfony\Component\HttpFoundation\Cookie(
-                'cid',
-                $request->query->get('cid'),
-                new \DateTime('+ 1 month')
-            )
-        );
+        $response = new RedirectResponse($url);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$response". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1488)